package io.github.cafeteriaguild.lin.rt.exceptions.internal

import io.github.cafeteriaguild.lin.rt.exceptions.LinException

class ContinueException : LinException("Generated continue signal")
